onnx_tf/handlers/frontend/size.py

```diff
@@ -0,0 +1,12 @@
+from onnx_tf.handlers.frontend_handler import FrontendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_op
+
+
+@onnx_op("Size")
+@tf_op("Size")
+class Size(FrontendHandler):
+
+  @classmethod
+  def version_9(cls, node, **kwargs):
+    return cls.make_node_from_tf_node(node, node.inputs[0], **kwargs)
```

Comment:
Default dtype of tf.size output is int32. We should cast to int64 to match onnx.
https://github.com/onnx/onnx/blob/master/docs/Operators.md#outputs-79

Reply:
Thanks for the comments. Changed the version to 1. But I don't know what needs to be done for the output dtype in the frontend handler. The backend handler was written before and already supports the output cast to int64 when calling tf.size. For the frontend handler, can't we simply ignore the dtype in tf.size (as we did for tf.shape) and allow onnx.size to return int64?

Reply:
We can't allow that. It will break the constraint.
For `shape`, it should be fixed.
Check the attr `out_type`, do nothing if it's `int64`, else, add `Cast` to cast it to `int32`.

Reply:
Got it. Added Cast to size frontend handler. Please review. If the code is okay, I can look into shape next.